gradle/libs.versions.toml

```diff
@@ -75,8 +75,7 @@ netty-buffer = "4.1.114.Final"
 netty-buffer-compat = "4.1.114.Final"
 object-client-bundle = "3.3.2"
 orc = "1.9.4"
-parquet = "1.13.1"
-pig = "0.17.0"
+parquet = "1.14.3"
```

Comment:
This looks like a merge conflict gone wrong. We've fallen back to 1.13.1 due to a bug in the dictionary encoding, we have https://github.com/apache/iceberg/pull/11502 to bump to `1.14.4`

Reply:
Fixed.